lib/src/client_browser.dart

```diff
@@ -115,6 +115,15 @@ class ClientBrowser extends ClientBase with ClientMixin {
     return this;
   }
 
+  @override
```

Comment:
Both the methods in client_io and client_browser are same they don't need specific access to io or browser features. This method should go into base class instead